lib/commands/exam.js

```diff
@@ -35,12 +35,14 @@ module.exports = TestCommand.extend({
   works: 'insideProject',
 
   availableOptions: [
-    { name: 'split',        type: Number,                  description: 'A number of files to split your tests across.' },
-    { name: 'partition',    type: [Array, Number],         description: 'The number of the partition(s) to run after splitting.' },
-    { name: 'weighted',     type: Boolean,                 description: 'Weights the type of tests to help equal splits.' },
-    { name: 'parallel',     type: Boolean, default: false, description: 'Runs your split tests on parallel child processes.' },
-    { name: 'load-balance', type: Number,                  description: 'Loads your test balanced' },
-    { name: 'random',       type: String,  default: false, description: 'Randomizes your modules and tests while running your test suite.' }
+    { name: 'split',        type: Number,                       description: 'A number of files to split your tests across.' },
+    { name: 'partition',    type: [Array, Number],              description: 'The number of the partition(s) to run after splitting.' },
+    { name: 'weighted',     type: Boolean,                      description: 'Weights the type of tests to help equal splits.' },
+    { name: 'parallel',     type: Boolean, default: false,      description: 'Runs your split tests on parallel child processes.' },
+    { name: 'load-balance', type: Number,                       description: 'Loads your test balanced' },
+    { name: 'random',       type: String,  default: false,      description: 'Randomizes your modules and tests while running your test suite.' },
+    { name: 'replay-execution', type: String,  default: false,  description: 'A JSON file path which maps from browser number to a list of modules'},
```

Comment:
For the commands that has longer string, we can add an alias to it:
`{ name: 'test-port',   type: Number,  default: 1234,     aliases: ['tp'], description: 'blah blah' },`
refer to https://github.com/ember-cli/ember-cli/blob/master/lib/commands/test.js